engine/src/main/java/org/terasology/engine/core/module/ModuleManager.java

```diff
@@ -79,8 +79,7 @@ public ModuleManager(String masterServerAddress, List<Class<?>> classesOnClasspa
 
         Map<String, ModuleMetadataLoader> mmlm = moduleFactory.getModuleMetadataLoaderMap();
         mmlm.put(TerasologyConstants.MODULE_INFO_FILENAME.toString(), metadataReader);
-        mmlm.put("engine-module.txt", metadataReader);  // FIXME: this should be *only* for engine-module.
-        // FIXME: …or maybe engine-module doesn't need a special filename anymore?
+        mmlm.put("module.txt", metadataReader);  // FIXME: this should be *only* for engine-module.
```

Comment:
I think `module.txt` is `MODULE_INFO_FILENAME`, so we can remove this second `mmlm.put`.